ui/server/src/test/scala/pl/touk/nussknacker/ui/api/PublicResourcesSpec.scala

```diff
@@ -0,0 +1,38 @@
+package pl.touk.nussknacker.ui.api
+
+import java.util.Collections
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import com.typesafe.config.ConfigFactory
+import io.circe.Json
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest._
+import pl.touk.nussknacker.engine.testing.{EmptyProcessConfigCreator, LocalModelData}
+import pl.touk.nussknacker.ui.api.helpers.EspItTest
+import pl.touk.nussknacker.ui.api.helpers.TestFactory.withoutPermissions
+
+class PublicResourcesSpec extends FunSuite with ScalatestRouteTest
+  with Matchers with ScalaFutures with OptionValues with BeforeAndAfterEach with BeforeAndAfterAll with EspItTest {
+
+  import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport._
+  import scala.collection.JavaConverters._
+  import io.circe.syntax._
+
+  lazy val creatorWithBuildInfo: EmptyProcessConfigCreator = new EmptyProcessConfigCreator {
+    override def buildInfo(): Map[String, String] = Map("fromModel" -> "value1")
+  }
+
+  val modelData: LocalModelData = LocalModelData(ConfigFactory.empty(), creatorWithBuildInfo)
+  val globalConfig: Map[String, String] = Map("testConfig" -> "testValue", "otherConfig" -> "otherValue")
+  val resources: PublicResources = new PublicResources(ConfigFactory.parseMap(Collections.singletonMap("globalBuildInfo", globalConfig.asJava)), Map("test1" -> modelData))
+
+  def routeResult(url: String): RouteTestResult = Get(url) ~> withoutPermissions(resources)
+
+  test("it should return buildInfo") {
```

Comment:
I know that you named class "Public..." but IMO would be better if add more descriptive info in test e.g. "It should return build info without authentication"

Reply:
Changed.